### PR TITLE
稳定性优化

### PR DESCRIPTION
**Pull request by solonobita**
1. 增加互联网搜集的cdn；
2. 修改心跳登录判定, 避免频繁提示掉线。

### CODE REVIEW

**Comment by solonobita** [comment]
目前测试二维码登录基本稳定，白天时段基本只弹出2~3次二维码登录请求；
密码登录个人账号基本不可用，初步怀疑是增加了滑动验证码识别，需要继续研究。

**Comment by solonobita** [comment]
滑动登录自测成功，需要广大网友多多测试

**Comment by solonobita** [comment]
Package                 Version
----------------------- --------------------
absl-py                 1.2.0
appdirs                 1.4.3
astunparse              1.6.3
attrs                   19.3.0
Automat                 0.8.0
autopep8                1.7.0
beautifulsoup4          4.7.0
blinker                 1.4
browser-cookie3         0.16.1
bs4                     0.0.1
cachetools              4.2.4
certifi                 2021.5.30
chardet                 3.0.4
charset-normalizer      2.1.1
Click                   7.0
cloud-init              22.2
colorama                0.4.3
command-not-found       0.3
configobj               5.0.6
constantly              15.1.0
cryptography            2.8
cssselect               1.0.3
dbus-python             1.2.16
DingtalkChatbot         1.3.0
distro                  1.4.0
distro-info             0.23ubuntu1
entrypoints             0.3
fake-useragent          0.1.11
Flask                   1.0.2
Flask-JWT-Extended      3.15.0
gast                    0.3.3
google-auth             1.35.0
google-auth-oauthlib    0.4.6
google-pasta            0.2.0
grpcio                  1.48.1
h5py                    2.10.0
httplib2                0.14.0
hyperlink               19.0.0
idna                    2.8
importlib-metadata      4.12.0
incremental             16.10.1
itsdangerous            1.1.0
Jinja2                  2.10
jsonpatch               1.22
jsonpointer             2.0
jsonschema              3.2.0
Keras                   2.4.0
Keras-Preprocessing     1.1.2
keyring                 18.0.1
language-selector       0.1
launchpadlib            1.10.13
lazr.restfulclient      0.14.2
lazr.uri                1.0.3
lightpush               0.1.3
lxml                    4.6.3
lz4                     4.0.2
Markdown                3.4.1
MarkupSafe              2.0.1
more-itertools          4.2.0
netifaces               0.10.4
numpy                   1.18.5
oauthlib                3.1.0
opencv-python           4.6.0.66
opt-einsum              3.3.0
p5py                    1.0.0
parse                   1.9.0
pbkdf2                  1.3
pbr                     5.10.0
pep517                  0.13.0
pexpect                 4.6.0
pip                     22.2.2
protobuf                3.9.2
pyaes                   1.6.1
pyasn1                  0.4.2
pyasn1-modules          0.2.1
pycodestyle             2.9.1
pycryptodome            3.15.0
pyee                    6.0.0
PyGObject               3.36.0
PyHamcrest              1.9.0
PyJWT                   1.7.1
pymacaroons             0.13.0
PyNaCl                  1.3.0
pyOpenSSL               19.0.0
pypng                   0.20220715.0
pyppeteer               0.0.25
pyppeteer-box           0.0.27
pyquery                 1.4.0
pyrsistent              0.15.5
pyserial                3.4
python-apt              2.0.0+ubuntu0.20.4.8
PyYAML                  5.3.1
redis                   3.0.1
requests                2.28.1
requests-html           0.9.0
requests-oauthlib       1.3.1
requests-unixsocket     0.2.0
rsa                     4.9
scipy                   1.4.1
SecretStorage           2.3.1
service-identity        18.1.0
setuptools              65.3.0
simplejson              3.16.0
six                     1.15.0
sos                     4.3
soupsieve               1.6.2
ssh-import-id           5.10
systemd-python          234
tensorboard             2.10.0
tensorboard-data-server 0.6.1
tensorboard-plugin-wit  1.8.1
tensorflow              2.3.0
tensorflow-estimator    2.3.0
termcolor               2.0.1
testresources           2.0.1
toml                    0.10.2
tomli                   2.0.1
tqdm                    4.64.1
Twisted                 18.9.0
typing_extensions       4.3.0
ubuntu-advantage-tools  27.10
ufw                     0.36
unattended-upgrades     0.1
urllib3                 1.26.12
w3lib                   1.19.0
wadllib                 1.3.3
websockets              7.0
Werkzeug                0.15.5
wheel                   0.37.1
wrapt                   1.14.1
zipp                    1.0.0
zope.interface          4.7.1

**Comment by IronXiao** [comment]
Linux使用滑动验证需要显示支持，如WSL 可运行于MobaXtrerm（带有一个X11 Server）

**Comment by pjialin** [comment]
感谢，看了下，应该没什么问题。里面有两个直接 `print` 的地方去掉一下会比较好。

**Comment by solonobita** [comment]
> 感谢，看了下，应该没什么问题。里面有两个直接 `print` 的地方去掉一下会比较好。

已经去掉了，调试完忘记删除了... ...
